Jenkinsfile

```diff
@@ -32,6 +32,19 @@ pipeline {
   }
 
   stages {
+    stage('Pull Request') {
```

Comment:
While there is a Pull Request stage in jenkins for this it isn't set up to do anything currently.

Reply:
this is a great addition!